include/seqan3/argument_parser/validators.hpp

```diff
@@ -600,8 +609,11 @@ class input_file_validator : public file_validator_base
 template <typename file_t = void>
 class output_file_validator : public file_validator_base
 {
-public:
+private:
+    //!\brief Stores the current mode of whether it is valid to overwrite the output file.
+    output_file_open_options mode;
```

Comment:
```suggestion
    output_file_open_options mode = output_file_open_options::create_new;
```


Let's define the default constructor properly :)